src/vs/workbench/contrib/notebook/common/model/notebookCellTextModel.ts

```diff
@@ -297,6 +297,15 @@ export class NotebookCellTextModel extends Disposable implements ICell {
 		} else {
 			output.replaceData(items);
 		}
+		if (output.outputs.every(item => isTextStreamMime(item.mime))) {
+			const mime = output.outputs[0].mime;
+			const compressed = compressOutputItemStreams(output.outputs[0].mime, output.outputs.map(item => item.data));
```

Comment:
TODO: This would be done based on a flag based on the original call via a proposed API